.packit.yaml

```diff
@@ -36,3 +33,6 @@ jobs:
     dist_git_branches:
     - el6
     - epel7
+  actions:
+    post-upstream-clone:
+    - rpmdev-bumpspec --comment='latest upstream build' ./packaging/epel/convert2rhel.spec
```

Comment:
@TomasTomecek, wouldn't the bumpspec apply only to the `propose_downstream` job?

Reply:
since this is the copr_build job, it will only apply to it, nothing else

if you inspect the link in my original post, you'll see there are actually 2 release bumps which is not ideal, but does the trick in the end

Reply:
Ok, it seemed to me the indentation of the `actions:` causes that it applies only to the `propose_downstream` job. But I'm no `yaml` guy, so I believe you that it applies to the whole config file :)
Or, could you perhaps move it to the line 5, below the `upstream_project_url`, so that nobody gets confused like I did?

Reply:
You're right, I put to a wrong job, my bad. Will fix and move as you suggest.
```
............................................________
....................................,.-'"...................``~.,
.............................,.-"..................................."-.,
.........................,/...............................................":,
.....................,?......................................................,
.................../...........................................................,}
................./......................................................,:`^`..}
.............../...................................................,:"........./
..............?.....__.........................................:`.........../
............./__.(....."~-,_..............................,:`........../
.........../(_...."~,_........"~,_....................,:`........_/
..........{.._$;_......"=,_......."-,_.......,.-~-,},.~";/....}
...........((.....*~_......."=-._......";,,./`..../"............../
...,,,___.`~,......"~.,....................`.....}............../
............(....`=-,,.......`........................(......;_,,-"
............/.`~,......`-...................................../
.............`~.*-,.....................................|,./.....,__
,,_..........}.>-._...................................|..............`=~-,
.....`=~-,__......`,.................................
...................`=~-,,.,...............................
................................`:,,...........................`..............__
.....................................`=-,...................,%`>--==``
........................................_..........._,-%.......`
..................................., 
```